pkg/trait/quarkus.go

```diff
@@ -47,6 +47,57 @@ type quarkusTrait struct {
 	BaseTrait
 	traitv1.QuarkusTrait `property:",squash"`
 }
+type languageSettings struct {
```

Comment:
This information would depend on the catalog. We should parse the catalog and use those values accordingly.

Reply:
I wanted initially to do that in a second step to keep it small and simple. But if you really want, I can make it dynamic right now?

Reply:
Whatever work best for you. Ideally it should be done before merging the PR to make sure we get this dynamically from the Catalog. Feel free to push a further commit to solve this at the very end of your development.

Reply:
ok I included it in my PR

Reply:
It is addressed now, I also added the support of legacy camel catalog